packages/runtime/test/consumption/attributes.test.ts

```diff
@@ -1110,140 +1110,150 @@ describe("Get (shared) versions of attribute", () => {
     let sOSRAVersion0: LocalAttributeDTO;
     let sOSRAVersion1: LocalAttributeDTO;
     let sOSRAVersion2: LocalAttributeDTO;
```

Comment:
I don't know when this was introduced, but WHAT IS THIS?! shouldn't those local variables be removed during decoupling the events?

cc @slavistan

Reply:
IC

Reply:
the helper methods also utilize this setup, are you sure you want this changed? (it would make the helper signatures larger)